libs/tasks/application/src/lib/tasks.service.ts

```
@@ -0,0 +1,36 @@
+import { Injectable } from '@nestjs/common';
+import { Task, UserTask } from '@tasks/domain';
+
+@Injectable()
+export class TasksService {
+  constructor() {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)